src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -32,29 +35,24 @@ const defaults = {
  */
 export default function applicantInformation(schema, options) {
   // Use the defaults as necessary, but override with the options given
-  const mergedOptions = _.assign(defaults, options);
+  const prefix = (options && options.isVeteran) ? 'veteran' : 'relative';
+  const mergedOptions = _.assign(defaults(prefix), options);
   const { fields, required, labels } = mergedOptions;
 
-  const possibleProperties = {
-    relativeFullName: schema.definitions.fullName,
-    relativeSocialSecurityNumber: schema.definitions.ssn,
+  const possibleProperties = _.assign(schema.properties, {
     'view:noSSN': {
       type: 'boolean'
-    },
-    vaFileNumber: schema.definitions.vaFileNumber,
-    relativeDateOfBirth: schema.definitions.date,
-    gender: schema.definitions.gender,
-    relationship: schema.definitions.relationship
-  };
+    }
+  });
 
   return {
     path: 'applicant/information',
     title: 'Applicant information',
     initialData: {},
     uiSchema: {
       'ui:order': fields,
-      relativeFullName: fullName.uiSchema,
-      relativeSocialSecurityNumber: _.assign(ssn.uiSchema, {
+      [`${prefix}FullName`]: fullName.uiSchema,
+      [`${prefix}SocialSecurityNumber`]: _.assign(ssn.uiSchema, {
```

Comment:
This SSN/noSSN/VA File number pattern is ubiquitous. I had moved it into `src/js/edu-benefits/definitions/veteranId.js` so it would stop cluttering our forms, but that was when I'd assumed it applied only to veterans. As I understand, it can also apply to non-veteran applicants who have previously received benefits. Should we create an issue to also refactor `veteranId.js` into a generic `personId`?

It could include these options
1. require either SSN or VA File Number
2. require neither, but the 'no ssn' checkbox must be checked if SSN is not available